includes/wikia/models/DesignSystemGlobalNavigationModelV2.class.php

```diff
@@ -29,15 +29,16 @@ public function __construct( $product, $productInstanceId, $lang = self::DEFAULT
 		$this->lang = $lang;
 	}
 
-	public function getData() {
+	public function getData( $isWikiaOrgCommunity = false ) {
```

Comment:
can't we pass isWikiaOrgCommunity to the constructor and use it as a class field?

Reply:
What kind of difference does it make?

Reply:
At least that would make code more consistent. We already pass related data to constructor:
```$this->product = $product;
$this->productInstanceId = $productInstanceId;	
$this->lang = $lang;```